The function clear_user_page() is defined in asm/page.h to be
__cpu_clear_user_page() which on the XScale architecture resolves to
xscale_mc_clear_user_page - which in turn needs to be exported for the
kernel modules (such as video-buf.ko) that use it.  Without these
exports, the video-buf kernel module emits the following warning:

WARNING: "xscale_mc_clear_user_page" [drivers/media/video/video-buf.ko] undefined!

Signed-off-by: Rod Whitby <rod@whitby.id.au>

 arch/arm/mm/copypage-xscale.c |    4 ++++
 1 files changed, 4 insertions(+)

Index: linux-2.6.17/arch/arm/mm/copypage-xscale.c
===================================================================
--- linux-2.6.17.orig/arch/arm/mm/copypage-xscale.c
+++ linux-2.6.17/arch/arm/mm/copypage-xscale.c
@@ -15,6 +15,7 @@
  */
 #include <linux/init.h>
 #include <linux/mm.h>
+#include <linux/module.h>
 
 #include <asm/page.h>
 #include <asm/pgtable.h>
@@ -129,3 +130,6 @@
 	.cpu_clear_user_page	= xscale_mc_clear_user_page, 
 	.cpu_copy_user_page	= xscale_mc_copy_user_page,
 };
+
+EXPORT_SYMBOL(xscale_mc_clear_user_page);
+EXPORT_SYMBOL(xscale_mc_copy_user_page);
